peru/runtime.py

```diff
@@ -139,6 +144,67 @@ def print_overrides(self):
             self.display.print('  {}: {}'.format(
                 name, self.get_override(name)))
 
+    def watch_overrides(self):
+        class WatchHandler(LoggingEventHandler):
+            logging.basicConfig(level=logging.INFO,
+                            format='%(asctime)s - %(message)s',
+                            datefmt='%Y-%m-%d %H:%M:%S')
+
+            def on_moved(self, event):
+                super(LoggingEventHandler, self).on_moved(event)
+                what = 'directory' if event.is_directory else 'file'
+                logging.info('moved %s: from %s to %s', what, event.src_path,
+                             event.dest_path)
+                logging.info('running peru sync --force')
+                os.system('peru sync --force')
```

Comment:
Rather than shelling out with `os.system`, we'd probably want to call directly into `imports.checkout` the way that `do_sync` does. That would mean that these functions would need to be coroutines.